ui.frontend/src/view/FormFieldBase.js

```diff
@@ -617,7 +637,7 @@ class FormFieldBase extends FormField {
                     pElement = document.createElement('p');
                     descriptionElement.appendChild(pElement);
                 }
-                pElement.textContent = sanitizedDescriptionText;
+                pElement.innerHTML = sanitizedDescriptionText;
```

Comment:
I think we need to improve this logic as follows, could you validate if this would work for all use-cases, including the migration bug which is discussed over slack


```
const sanitizedDescriptionText = window.DOMPurify ? window.DOMPurify.sanitize(descriptionText) : descriptionText;
let descriptionElement = this.getDescription();

if (descriptionElement) {
    // Check if the content inside the descriptionElement needs updating
    let currentTextContent = descriptionElement.innerText.trim();

    if (currentTextContent === sanitizedDescriptionText) {
        // No update needed if the text content already matches
        return;
    }

    // Find the existing <p> element
    let pElement = descriptionElement.querySelector('p');

    if (!pElement) {
        // If no <p> tag exists, create one and set it as the content
        pElement = document.createElement('p');
        descriptionElement.innerHTML = ''; // Clear existing content
        descriptionElement.appendChild(pElement);
    }

    // Update the <p> element's content with sanitized content
    pElement.innerHTML = sanitizedDescriptionText;
}


```